beta/src/pages/learn/thinking-in-react.md

```diff
@@ -2,17 +2,18 @@
 title: Thinking in React
```

Comment:
`Thinking in React` we need to translate this. only `title` keyword will not be translated

Reply:
`React में सोच` => `React में सोचें`